suite-native/app/package.json

```diff
@@ -1,42 +1,26 @@
 {
-    "name": "@trezor/suite-native",
+    "name": "@suite-native/app",
     "version": "1.0.0",
     "suiteNativeVersion": "24.2.0",
-    "private": true,
+    "main": "index.js",
     "scripts": {
-        "android": "react-native run-android --no-packager --mode=devDebug --appIdSuffix=debug --active-arch-only",
+        "android": "expo run:android",
         "android:clean": "cd android && ./gradlew clean",
-        "ios": "react-native run-ios --no-packager --scheme \"Debug\"",
-        "start": "react-native start --reset-cache",
-        "test": "jest",
+        "ios": "expo run:ios",
+        "start": "expo start --dev-client",
         "lint:js": "yarn g:eslint .",
         "type-check": "tsc -b",
-        "pods": "npx pod-install"
-    },
-    "expo": {
-        "autolinking": {
-            "exclude": [
-                "expo-keep-awake",
-                "expo-font",
-                "expo-file-system"
-            ]
-        }
+        "pods": "expo prebuild --clean"
```

Comment:
better to rename? `prebuild`?

Reply:
I reworked scripts, since `pods` command could be useful I reverted it and added prebuild commands